Lab9.py

```diff
@@ -0,0 +1,20 @@
+class Accounting:
+    def __init__(self):
+        self.products = []
+
+
+class Product(Accounting):
+    def __init__(self, product_id):
+        super().__init__()
+        self.id = product_id
+
+
+class Customer(Product):
+    def __init__(self, name, product_id):
+        super().__init__(product_id)
+        self.name = name
+
+
+accounting = Accounting()
+cola = Product(105)
+ivan = Customer("Ivan", 105)
```

Comment:
Maybe print some information on your objects? As the moment, your program doesn't really do much.
To do a proper print, consider reading about [`__str__` method](https://realpython.com/lessons/how-and-when-use-str/).